tests/resolve-relative-imports.test.ts

```diff
@@ -0,0 +1,25 @@
+import { resolveFilePath } from "lib/runner/resolveFilePath"
+import { expect, test } from "bun:test"
+
+test("resolveFilePath should handle relative imports from subdirectories", () => {
+  const mockFs = {
+    "components/button.tsx": "export const Button = () => <button />",
+    "components/shared/icons/icon.tsx": "export const Icon = () => <svg />",
+    "components/shared/utils.ts": "export const formatText = (t: string) => t",
```

Comment:
Make this relevant to tscircuit

Reply:
Also the test name doesnt match the directory pattern if im not mistaken, the test name should match

Reply:
Also it should be an e2e test rather than a unit test